src/components/ShowBooks.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Book from './Book';
+import Form from './Form';
+
+const ShowBooks = () => (
+  <div>
+    <Book title="Book 1" />
+    <Book title="Book 2" />
+    <Form />
```

Comment:
- [ ] [OPTIONAL] I would highly recommend that instead calling `book` component manually, use an array to store the book objects, each with a title and author and use  `map` to iterate over the array, in order to make the most out of reusable components